pyproject.toml

```diff
@@ -2,7 +2,7 @@
 requires = [
   "wheel",
   "setuptools",
-  "torch>=1.7",
+  "torch>=1.10",
```

Comment:
Readme note should be alone good for now.. we can keep others sync with monai core..